src/plugins/discover/public/application/angular/doc_table/doc_table.ts

```diff
@@ -63,7 +65,7 @@ export function createDocTableDirective(pagerFactory: any, $filter: any) {
         $scope.limit = Math.max(minimumVisibleRows || 50, $scope.limit || 50);
       });
 
-      $scope.$watch('hits', (hits: any) => {
+      $scope.$watch('hits', (hits: unknown[]) => {
```

Comment:
Would `ElasticsearchHit` as a type not work here?

Reply:
Yes and no. It's the correct type, but it will require us to go a bit  deeper in the typing and change more things across the whole plugin to get to a proper typed situation. I did not want to "fix" all the typings in this PR, since I wanted to keep it manageable and quickly mergable for now. A lot of those places still need proper cleanup (I'd say nearly all the `unknown` can use proper typings. But especially the usage of `ElasticsearchHit` and proper request/response typing will need more (further) PRs to cleanup.